back/db/entrypoint/init.sql

```diff
@@ -11,3 +11,10 @@ CREATE TABLE IF NOT EXISTS news (
     created_date DATE NOT NULL,
     modified_date DATE NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS users (
+    uid         INT             AUTO_INCREMENT  PRIMARY KEY,
+    email       VARCHAR(64)     NOT NULL        UNIQUE,
+    password    TEXT            NOT NULL,
+    nickname    TEXT            NOT NULL
```

Comment:
I think `name` is better. Because in the front end, we use `name` rather than `nickname`.

Reply:
Both ok. But I think nickname better.

Reply:
:eyes: I can change the front end. What do you think?

Reply:
My idea is to distinguish logins by nicknames.
If you can distinguish, it's up to you to decide

Reply:
OK. Get it.